src/igvfd/types/sample.py

```diff
@@ -491,10 +491,18 @@ def summary(self, request, sample_terms, donors, sex, age, age_units=None, embry
             if perturbation_treatment_summaries:
                 summary_terms += f' treated with {", ".join(perturbation_treatment_summaries)},'
 
+        # modification summaries are appended to the end of the summary
+        if (modifications and
+                biosample_type in ['primary_cell', 'in_vitro_system', 'tissue', 'whole_organism']):
+            modification_objects = [request.embed(modification) for modification in modifications]
+            modification_summaries = sorted([modification.get('summary') for modification in modification_objects])
+            if modification_summaries:
+                summary_terms += f' modified with {", ".join(modification_summaries)},'
+
         # construct library set overview is appended to the end of the summary
         if (construct_library_sets and
                 biosample_type in ['primary_cell', 'in_vitro_system', 'tissue', 'whole_organism']):
-            verb = 'modified with'
+            verb = 'transfected with'
             library_types = set()
             for CLS in construct_library_sets:
```

Comment:
Slightly out of scope but since we are editing this code: can we change this variable from "CLS" to something else?  I think "CLS" is a special reserved keyword in Python (similar to "self") so we probably shouldn't use it if we can avoid it. 
I think this is also used in the Technical Sample code as well.

Reply:
fixed